Classes/RMRemoteLogger.h

```diff
@@ -0,0 +1,68 @@
+//
+//  RMRemoteLogger.h
+//
+
+#import <Foundation/Foundation.h>
+
+#import <CocoaLumberjack/CocoaLumberjack.h>
+
+@interface RMRemoteLogger : DDAbstractLogger {
```

Comment:
This class is a direct copy of RMPaperTrailLogger. Renamed it so it's clear that it's a generic remote logger when we use it for data dog. This prevents confusion by the the users of this pod. Did not delete RMPaperTrailLogger because our other internal pods may still use that class and I do not want to break things.

Reply:
@jnhuynh Should we have `RMPapertrailLogger` extend this class with no extra functionality? Just to make it clear that it's the same thing, just a different class name.

Reply:
Not a bad idea.

Reply:
@andrewkatz any other feedback or can I merge this?